spec/jobs/chat_channel_delete_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
```suggestion
```

Btw this is no longer required in all spec files going forward. See https://github.com/discourse/discourse/commit/c9dab6fd087dd400c904441657f53134393fb298.

Reply:
Ooh nice!

Reply:
Would be nice to have changes like this posted on dev tbh, if you're not watching the core commits you're likely to miss stuff like this.